contents/docs/data/persons.mdx

```diff
@@ -16,18 +16,21 @@ export const personProfileDark = "https://res.cloudinary.com/dmukukwp6/image/upl
     className="rounded shadow-xl"
 />
 
-People in PostHog represent users who do events.
+People in PostHog represent the users behind your events. You can view them in the [People tab](https://us.posthog.com/persons).
 
-Every event has a `distinct_id` representing a person attached to it. However, not every `distinct_id` has an associated person profile.
+People have [user properties](/docs/product-analytics/user-properties), which enables you do things like:
 
-Capturing person profiles and user properties enables:
+- Filter on user properties
+- Create [cohorts](/docs/data/cohorts)
+- Use user properties for targeting feature flags, A/B tests, and surveys.
```

Comment:
```suggestion
- Target feature flags, A/B tests, and surveys.
```

Reply:
I will keep as is, since "Target feature flags, A/B tests, and surveys." makes it sound like you're not able to target at all with user properties (you can still target on other things)